trmem.c

```diff
@@ -32,9 +32,9 @@
 #include <stdarg.h>
 #include <stdio.h>
 #include <stdlib.h>
-#include <string.h>
+//#include <string.h>
 #include <ctype.h>
-
+#include <errno.h>

```

Comment:
Does this include really needed?

Reply:
errno was throwing error on MacOSX builds as errno was undeclared, so this solves this error : 

```
JWasm/trmem.c:783:69: error: use of undeclared identifier 'errno'
        printf("tm_Init: fopen(\"" TRMEM_LOGFN "\") failed [%u]\n", errno );
```

Reply:
Had no such error on my OSX with`make -f GccUnix.mak`...

Reply:
I see where confusion came from :) in GccUnix.mac which includes from gccmod.inc doesn't use trmem.c, if you add trmem.c to gccmod.inc as '$(OUTD)/trmem.o' it will cause error on Mac. In cmake file I populate all .c files and compile them, hence trmem.c is also compiled and will produce error. So these changes can be excluded, I will update CMakeLists.txt to exclude trmem.c, as it's never used.